### DISCUSSION TITLE
Adding frame/components name?

### DISCUSSION THREAD

**Discussion by Hideman85** [pocka/figspec#42] [Ideas]
Hi,

First of all this project is awesome, the default embed is so inconvenient... and this is the solution :rocket: 

I'm wondering if the preview can also have the naming that we can see on the original file? For the development experience this is so much better to have that kind of metadata directly visible :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/16165119/169273656-b9643cb2-2153-4779-a96b-8037e2489024.png)

Thanks in advance for your help :wink:

**Comment by pocka** [comment]
I think this isn't difficult to implement because both `COMPONENT` and `FRAME` nodes already have `name` property so the renderer only needs to calculate the label position and create a DOM node for it. I'm not sure how other top-level things such as orphan instances will end up in an API response, though.
